src/components/Header/Logo/logo.jsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import mylogo from "./../../../assets/unnamed.png";
+function Logo() {
```

Comment:
Try to make arrow functions as much as possible
```suggestion
const Logo = () => {
```